ckanext/dcat/processors.py

```diff
@@ -146,13 +152,123 @@ def datasets(self):
             yield dataset_dict
 
 
+class RDFSerializer(RDFProcessor):
+    '''
+    A CKAN to RDF serializer based on rdflib
+
+    Supports different profiles which are the ones that will generate
+    the RDF graph.
+    '''
+
+    def graph_from_dataset(self, dataset_dict):
+        '''
+        Given a CKAN dataset dict, creates a graph using the loaded profiles
+
+        The class RDFLib graph (accessible via `parser.g`) will be updated by
```

Comment:
you are right, forgot to update the docstrings on b2887d79b28bad634172b3f4ee657f65f793b3c2